rust_src/src/data.rs

```diff
@@ -581,7 +581,7 @@ pub unsafe extern "C" fn store_symval_forwarding(
                     prop = get(pred_sym, Qrange);
                     if let Some((min, max)) = prop.into() {
                         let args = [min, newval, max];
-                        if !newval.is_number() || leq(&args) {
+                        if !newval.is_number() || !leq(&args) {
```

Comment:
Good find. The Emacs source does not make use of deMorgan's much. Is it worth it here?
```suggestion
                        if !(newval.is_number() && leq(&args)) {
```

Reply:
I don't think there's really any reason to. I've always thought that it reads better this way.